packages/salesforcedx-vscode-core/src/commands/baseDeployCommand.ts

```diff
@@ -60,20 +60,17 @@ export abstract class BaseDeployExecutor extends SfdxCommandletExecutor<
     });
 
     execution.processExitSubject.subscribe(async exitCode => {
-      let properties;
-      try {
-        const ws = ComponentSet.fromSource(execFilePathOrPaths);
-        const metadataCount = JSON.stringify(createComponentCount([...ws]));
-        properties = { metadataCount };
-        // registry does not handle multiple paths. only log component count for single paths
-      } catch (e) {
-        telemetryService.sendException(
-          e.name,
-          'error detecting deploy components'
-        );
+      const telemetry = new TelemetryBuilder();
+      if (execFilePathOrPaths) {
```

Comment:
Let's add this on a try/catch. Looking at the telemetry this code generates the most common exception.

Reply:
Yeah I'll add that back in. That exception count should drop a decent amount after this change since we're handling multiple paths now.

Reply:
@brpowell should we tweak the exception info a bit ? Right now it's impossible to know what caused it.